src/client/index.js.wrapper.mustache

```diff
@@ -1,3 +1,5 @@
 (function initHammerheadClient () {
     {{{source}}}
 })();
+
+window['%hammerhead%'].utils.removeInjectedHeadScript();
```

Comment:
I think that the `head` word is extra. Just use the `removeInjectedScript` function name.